EvidenceApi/V1/Controllers/DocumentSubmissionsController.cs

```diff
@@ -58,6 +61,32 @@ public async Task<IActionResult> UpdateDocumentSubmissionState([FromRoute][Requi
             }
         }
 
+        /// <summary>
+        /// Updates the visibility of a document submission
+        /// </summary>
+        /// <response code="200">Updated</response>
+        /// <response code="400">Request contains invalid parameters</response>
+        /// <response code="404">Document submission cannot be found</response>
+        [HttpPatch]
+        [Route("{id}/visibility")]
+        public IActionResult UpdateDocumentSubmissionVisibility([FromRoute][Required] Guid id, [FromBody] DocumentSubmissionVisibilityUpdateRequest request)
```

Comment:
Would be good to add a test that confirms the required id check works.

Reply:
I am struggling to create a test for this, as the Guid cannot be passed as null to the class under test.
Open to suggestions.

Reply:
One option with the current setup would be to add e2e test that calls `var uri = new Uri($"api/v1/document_submissions/{createdDocumentSubmission.Id}/visibility", UriKind.Relative);` without the `createdDocumentSubmission.Id` and then test that it returns bad request. Similar setup to `CanUpdateDocumentSubmissionVisibilityWithValidParameters` test.